bundle-macos-app.ps1

```diff
@@ -0,0 +1,80 @@
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$MatrixAssetName,
```

Comment:
Let's rename this to something specific to the bundling process and not the CI/CD process

```suggestion
    [string]$BundleName,
```